src/DiscordMember.js

```diff
@@ -88,9 +88,11 @@ class DiscordMember {
           nicknameData.groupRank = groups.Role
           break
         } else {
-          nicknameData.groupRank = ' '
+          nicknameData.groupRank = 'Guest'
         }
       }
+    } else {
+      nicknameData.groupRank = ' '
```

Comment:
No need for this to be set, or for groupRank to even appear on the nicknameData on line 78